src/ui/components/results/verticalresultscomponent.js

```diff
@@ -227,7 +227,7 @@ export default class VerticalResultsComponent extends Component {
 
   getVerticalURL (data = {}) {
     if ('verticalURL' in this._config) {
-      return this._config.verticalURL;
+      return addParamsToUrl(this._config.verticalURL || '', { query: this.query });
```

Comment:
Iif someone were to give `url` as undefined, null, or `''` in the universalresults, this would save them and give them an empty (ie index.html) url.

Is this the behavior we want?

Reply:
I'll ask Rose exactly what we want the behavior to be when verticalURL is set vs unset vs '' vs null vs undefined, and how it should default. I probably shouldn't have tried to come up with conclusions on my own on how this config option should work.

Reply:
Updated it to not try to be cute with the `in` check. Will wait for Rose's response

Reply:
sorry dodging the original question, no I don't think we want that defaulting behavior

Reply:
Rose said for the defaulting behavior just be consistent with the current code